### PR TITLE
Add pre commit hook loader and pre commit rubocop

### PR DESCRIPTION
**Pull request by go8soft**
**Background**
We are using `rubocop` for code style.

**Aim**
We want to validate file changes style with rubocop before commit.

**Implementation**
Add pre-commit `rubocop` hook.

**Notes**
To set git hooks new dir:
```
i-have-i-need$ git config core.hooksPath .githooks
```

Check for X permissions. Here is what you need:
```
i-have-i-need/.githooks$ chmod +x pre-commit
i-have-i-need/.githooks$ chmod +x pre-commit.d/01-rubocop
```

Test it. Try to commit file with `rubocop` error.

We need to add this hooks instructions to README.

### CODE REVIEW

**Comment by go8soft** [comment]
> You could just update the readme as part of this PR?

Ok, I will

**Comment by go8soft** [comment]
> > You could just update the readme as part of this PR?
> 
> Ok, I will

Done

File: README.md [hunk 1]
```diff
@@ -40,6 +40,13 @@ docker-compose up -d
 rails db:setup
 
 rails server
+
+# to set git hooks
```
Comment (eddieleeper-nb) [inline README.md:44]: Could this be moved into it's own section under teh heading Git Hooks.

Comment (go8soft) [reply README.md]: Done: https://github.com/wearefuturegov/beacon/pull/101/commits/a9fa4989afbe0e9b4ff0639e8eed5ca77f5e4c77

**Pull request merged**